### PR TITLE
add-command-set-platform

### PR DESCRIPTION
**Pull request by KiwiUserSFTW**
add new commands: set-platform
add new hooks: usePlatform

### CODE REVIEW

File: src/commands/basic/set-platform.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+const usePlatform = require('@hooks/usePlatform');
+const [getCurrentPlatforms, getSupportedPlatforms, setPlatform] = usePlatform();
+const { SlashCommandBuilder } = require('@discordjs/builders');
+const platform_list = getSupportedPlatforms();
+module.exports = {
+    data: new SlashCommandBuilder()
+        .setName('set-platform')
+        .setDescription('What platform you want choose ?')
+        .addStringOption(option =>
+            option.setName('name')
```
Comment (gdivakov) [inline src/commands/basic/set-platform.js]: platform

File: src/commands/basic/set-platform.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        
+        await interaction.respond(
+           platform_list.map( title => ({ name: title, value: title }))
+        );
+    },
+    async execute({ interaction }) {
+        const platformQuery = interaction.options.getString('name');
+
+        if (!setPlatform(platformQuery)) {
+            return await interaction.reply('This platform isn`t support')
```
Comment (gdivakov) [inline src/commands/basic/set-platform.js]: isn't supported

Comment (KiwiUserSFTW) [reply src/commands/basic/set-platform.js]: done

File: src/commands/basic/set-platform.js (hunk 1 above)
Comment (KiwiUserSFTW) [reply src/commands/basic/set-platform.js]: done

**Pull request closed without merging**